feature/session/src/main/java/io/github/droidkaigi/confsched2020/session/ui/SearchSessionsFragment.kt

```diff
@@ -100,10 +94,14 @@ class SearchSessionsFragment : DaggerFragment() {
 
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
+        FragmentSearchSessionsBinding.bind(view).setup()
+    }
+
+    private fun FragmentSearchSessionsBinding.setup() {
```

Comment:
I agree with the overall policy!
This is my preference, but since there is not much processing other than binding, I prefer not to split this method.

Reply:
Thanks for agreeing.
Do you mean refactor like the following?
```kotlin
    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
        super.onViewCreated(view, savedInstanceState)
        val binding = FragmentSearchSessionsBinding.bind(view)
        ...
        binding.searchSessionRecycler.adapter = groupAdapter
        ...
    }
```